README.md

```diff
@@ -31,6 +32,9 @@ Local-Storage (LS) provides a cloud-native local storage system. It aims to prov
 ### Scheduler
 The Scheduler is to automatically schedule the Pod to the correct node which has the associated HwameiStor volume.
 
+### Admission-Controller
+Admission-Controller is a webhook that can automatically determine the pod that uses the HwameiStor Volume and help it modify the schedulerName to hwameistor-scheduler.
```

Comment:
```suggestion
Admission-Controller is a webhook that can automatically determine which pod uses the HwameiStor volume and, help to modify the schedulerName to hwameistor-scheduler.
```
Could we give a consistent word for `Admission-Controller`. Otherwise, it may contains many variants later that will confuse most people:
- admission-controller
- Admission-Controller
- Admission Controller
- Admission controller
Maybe we should define a glossary list.